src/style.css

```
@@ -1 +1,78 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header_wrapper {
+  width: 95%;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo {
+  flex-grow: 6;
+}
+.logo__link {
+  display: inline-block;
+}
+.logo__img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav{
+  flex-grow: 1;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav__item {
+  list-style: none;
+  text-transform: uppercase;
+  margin-right: 10px;
```

```suggestion
  margin-right: 20px;
```
По дизайну відступи між елементами мають бути 20px